server/db/recreate-schema.sql

```diff
@@ -1,9 +1,32 @@
--- Drop tables in case they already exist
+DROP TABLE if exists documents;
+DROP TABLE if exists languages;
```

Comment:
You will also need to drop the type you define below, otherwise next time you run `npm run recreate-db:local` it will fail. You can use `DROP TYPE user_role` (and same for document_status).

Reply:
it is done